app/chitter.rb

```diff
@@ -0,0 +1,102 @@
+require_relative 'data_mapper_setup'
+require 'sinatra/base'
+require 'sinatra/flash'
+require 'dm-timestamps'
+
+class Chitter < Sinatra::Base
+  use Rack::MethodOverride
+  enable :sessions
+  set :sessions_secret, 'super secret'
+  register Sinatra::Flash
+
+  helpers do
+    def current_user
+      @current_user ||= User.get(session[:user_id])
+    end
+  end
+
+  get '/' do
+    redirect '/peeps'
+  end
+
+  get '/peeps' do
+    p Peeps.all
+    @peeps = Peeps.all.sort_by{|peep| peep.created_at}
```

Comment:
Style/SymbolProc: Pass `&:created_at` as an argument to `sort_by` instead of a block.